docs/source/AdministratorGuide/Resources/computingelements.rst

```diff
@@ -99,8 +99,7 @@ of the *ComputingElement* is located in the inside the corresponding site sectio
 This is the general structure in which specific CE descriptions are inserted.
 The CE configuration is part of the general DIRAC configuration
 It can be placed in the general Configuration Service or in the local configuration of the DIRAC installation.
-
-Additional info can be found :ref:`here <resourcesComputing>`.
+Examples of the configuration can be found in the `Full Configuration Example`_, in the *Resources/Computing* section.
```

Comment:
@andresailer is it this the reason why the documentation build fails? I can't see the actual error.

Reply:
Yes, the error is in the logs
```
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Resources/computingelements.rst:99: ERROR: Unknown target name: "full configuration example".
```
You need to scroll to the right to see it 
This needs the sphinx.ext.autosectionlabel extension to be enabled? 
https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html#module-sphinx.ext.autosectionlabel

Reply:
No idea. How would we try?

Reply:
add the extension to the list
https://github.com/DIRACGrid/DIRAC/blob/0696d01e08f417cdde1c877fec40e4122a2642d7/docs/source/conf.py#L104-L108

Reply:
It did not work

Reply:
This extension causes a lot of warnings by itself, so better manually add the label for the Full Configuration Example and refer to that, via the script that creates that file.

Reply:
```
[snip]
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/FTS3Agent.rst:2: WARNING: duplicate label fts3agent, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/DataManagement/fts3.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/MoveReplica.rst:2: WARNING: duplicate label movereplica, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/PutAndRegister.rst:2: WARNING: duplicate label putandregister, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/RegisterFile.rst:2: WARNING: duplicate label registerfile, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/RegisterReplica.rst:2: WARNING: duplicate label registerreplica, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/RemoveFile.rst:2: WARNING: duplicate label removefile, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/RemoveReplica.rst:2: WARNING: duplicate label removereplica, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.rst:2: WARNING: duplicate label replicateandregister, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/AdministratorGuide/Systems/RequestManagement/rmsObjects.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/RequestOperations_Module.rst:10: WARNING: duplicate label modules, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/Agent_Module.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Client/Client_Module.rst:2: WARNING: duplicate label client, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/Core/Tornado/Client/Client_Module.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Client/Client_Module.rst:10: WARNING: duplicate label subpackages, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/Agent_Module.rst
/home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Client/Client_Module.rst:18: WARNING: duplicate label modules, other instance in /home/docs/checkouts/readthedocs.org/user_builds/dirac/checkouts/5847/docs/source/CodeDocumentation/DataManagementSystem/Agent/RequestOperations/RequestOperations_Module.rst
[snip]
```